### DISCUSSION TITLE
[BUG] Unable to upload additional proof

### DISCUSSION THREAD

**Discussion by Kira-Pgr** [community/community#68158] [GitHub Education]
### Hi everyone,

The application stays on the `Upload proof of your academic status` Page forever
[Video of repuducing the probem](https://1drv.ms/v/s!AuHkD4Yx8E4T5AzVn7ufbG9LlpPy?e=REcU4Z)
I've tried using different broswers and devices, still the same

**Comment by harith1996** [comment]
Same issue here. I've tried in incognito mode with two different browsers (chrome and firefox). 
The page still reloads. :(

**Comment by Amsinkamsink** [comment]
Yes, I have the same situation using Google and Safari,There is a high probability that it is a GitHub problem and will be resolved soon.
